src/autopas/containers/verletClusterLists/VerletClusterCells.h

```diff
@@ -97,8 +97,10 @@ class VerletClusterCells : public ParticleContainer<FullParticleCell<Particle>>
   void addParticleImpl(const Particle &p) override {
     if (autopas::utils::inBox(p.getR(), this->getBoxMin(), this->getBoxMax())) {
       _isValid = false;
-      // removes dummy particles in first cell
-      this->_cells[0].resize(_dummyStarts[0]);
+      // removes dummy particles in first cell, if the cell is not empty!
+      if (this->_cells[0].begin().isValid()) {
+        this->_cells[0].resize(_dummyStarts[0], *(this->_cells[0].begin()));
+      }
```

Comment:
this changes the behavior a bit:
now no dummy particles are generated if a cell contains no particles.